src/config.ts

```diff
@@ -1,71 +1,73 @@
 import deepFreeze from 'deep-freeze';
 import { UNIT_TIME, MILLISECONDS } from './types/Common';
 
-export const config = () => {
-  return deepFreeze({
-    version: '0.1',
-    /** Configuration related with session */
-    session: {
-      /** TimeOut in Minutes default 24 hours */
-      sessionTimeOut: 24 * UNIT_TIME.HOUR * MILLISECONDS,
-      /** Extended Session in Minutes default 30 minutes */
-      extendedSessionTimeOut: 30 * UNIT_TIME.MINUTE * MILLISECONDS,
-      /** TimeOut Session remain for warning user default 1 minute */
-      timeOutforWarningUser: 1 * UNIT_TIME.MINUTE * MILLISECONDS
+const conf = {
+  version: '0.1',
+  /** Configuration related with session */
+  session: {
+    /** TimeOut in Minutes default 24 hours */
+    sessionTimeOut: 24 * UNIT_TIME.HOUR * MILLISECONDS,
+    /** Extended Session in Minutes default 30 minutes */
+    extendedSessionTimeOut: 30 * UNIT_TIME.MINUTE * MILLISECONDS,
+    /** TimeOut Session remain for warning user default 1 minute */
+    timeOutforWarningUser: 1 * UNIT_TIME.MINUTE * MILLISECONDS
+  },
+  /** Toolbar Configuration */
```

Comment:
As the diff won't tell you, I tell you: there's no change in this whole object :-)